tests/test.sh

```diff
@@ -1,16 +1,26 @@
 #!/bin/bash
 
 echo "IO testing"
-cd IO && python IO_test.py -vv && cd ..
+cd IO
+python3 IO_test.py -vv || true
```

Comment:
If the tests fail, would this still print an error?

Reply:
Yes, it still prints the errors.  It just allows it to continue after that.  Here's an example:

```
2021-07-07T18:49:06[DEBUG   ] IOROOTProcessor(151) -> Creating branches
2021-07-07T18:49:06[DEBUG   ] IOROOTProcessor(166) -> Adding data
2021-07-07T18:49:06[DEBUG   ] IOROOTProcessor(180) -> File saved!
2021-07-07T18:49:06[INFO    ] BaseProcessor(78) -> Done with <WriterROOT>
2021-07-07T18:49:06[INFO    ] BaseProcessor(74) -> Run <ReaderROOT>...
2021-07-07T18:49:06[DEBUG   ] IOROOTProcessor(51) -> Reading myTest.root
2021-07-07T18:49:07[WARNING ] IOROOTProcessor(62) -> An uproot related error was encountered. Switching to ROOT.
ERROR

======================================================================
ERROR: test_ROOTIO (__main__.IOTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/p8/morpho/v2.7.1/lib/python3.8/site-packages/morpho/processors/IO/IOROOTProcessor.py", line 58, in Reader
    for key, value in data.items():
  File "/usr/local/lib/python3.8/dist-packages/awkward/highlevel.py", line 1125, in __getattr__
    raise AttributeError(
AttributeError: no field named 'items'

(https://github.com/scikit-hep/awkward-1.0/blob/1.4.0/src/awkward/highlevel.py#L1127)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "IO_test.py", line 107, in test_ROOTIO
    b.Run()
  File "/usr/local/p8/morpho/v2.7.1/lib/python3.8/site-packages/morpho/processors/BaseProcessor.py", line 75, in Run
    if not self.InternalRun():
  File "/usr/local/p8/morpho/v2.7.1/lib/python3.8/site-packages/morpho/processors/IO/IOProcessor.py", line 67, in InternalRun
    return self.Reader()
  File "/usr/local/p8/morpho/v2.7.1/lib/python3.8/site-packages/morpho/processors/IO/IOROOTProcessor.py", line 76, in Reader
    {str(varName): self.data[str(varName)] + value})
TypeError: can only concatenate list (not "cppyy.LowLevelView") to list

----------------------------------------------------------------------
Ran 4 tests in 0.423s

FAILED (errors=1)
Misc testing

RooFit v3.60 -- Developed by Wouter Verkerke and David Kirkby 
                Copyright (C) 2000-2013 NIKHEF, University of California & Stanford University
                All rights reserved, please read http://roofit.sourceforge.net/license.txt

test_ProcAssistant (__main__.MiscTests) ... 2021-07-07T18:49:09[INFO    ] misc_test(15) -> Processor Assistant test
2021-07-07T18:49:09[DEBUG   ] BaseProcessor(38) -> Creating processor <assistantProcessor>
2021-07-07T18:49:09[INFO    ] BaseProcessor(52) -> Configure <assistantProcessor>
2021-07-07T18:49:09[INFO    ] ProcessorAssistant(49) -> Found myFunction using myModule
2021-07-07T18:49:09[INFO    ] BaseProcessor(74) -> Run <assistantProcessor>...
2021-07-07T18:49:09[INFO    ] myModule(8) -> This is my function
2021-07-07T18:49:09[INFO    ] myModule(9) -> I will return: value=10
2021-07-07T18:49:09[INFO    ] BaseProcessor(78) -> Done with <assistantProcessor>
2021-07-07T18:49:09[DEBUG   ] misc_test(25) -> Assistant processor returned: value=10
ok

----------------------------------------------------------------------
Ran 1 test in 0.017s

OK
```

Reply:
I see thanks!